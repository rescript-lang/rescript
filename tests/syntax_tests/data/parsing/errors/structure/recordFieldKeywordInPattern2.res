let {a: _, type: x} = r
