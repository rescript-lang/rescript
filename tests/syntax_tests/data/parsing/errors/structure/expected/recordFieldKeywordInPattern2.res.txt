
  Syntax error!
  syntax_tests/data/parsing/errors/structure/recordFieldKeywordInPattern2.res:1:12-15

  1 │ let {a: _, type: x} = r
  2 │ 

  Cannot use keyword `type` here. Keywords are not allowed as record field names.

let { a = _; type_ = x } = r